src/functionalTest/resources/features/F-007 - Create CIVIL Staff and Judicial Role Assignments/F-007.feature

```diff
@@ -322,7 +322,7 @@ Feature: F-007 : Create Role Assignments for CIVIL Caseworker and Judicial Users
     And a successful call [to delete existing role assignments corresponding to the test actorId] as in [DeleteDataForRoleAssignments].
 
   @S-007.06_AAA6
-  @FeatureToggle(DB:civil_wa_2_1=on) @FeatureToggle(EV:CASEWORKER_FTA_ENABLED=on)
+  @FeatureToggle(DB:civil_wa_2_2=on) @FeatureToggle(EV:CASEWORKER_FTA_ENABLED=on)
```

Comment:
Can you revert the `FeatureToggle` change to the follwoig scenarios as these mappings are not under change in this ticket:
* `@S-007.06_AAA6`
* `@S-007.06_AAA7`
* `@S-007.06a_AAA6`
* `@S-007.06a_AAA7`
* `@S-007.06b_AAA6`
* `@S-007.06b_AAA7`

Reply:
Resolved in 676cb73a8562b8d41de042624b5163e151bb3589.